### PR TITLE
[office-js][office-js-preview] (Outlook) Remove preview string from new Outlook on Windows references

### PR DESCRIPTION
**Pull request by samantharamon**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:

- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: None.
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -15736,16 +15792,17 @@ declare namespace Office {
...
          * To determine the Outlook client in which your add-in is running, use the `mailbox.diagnostics.hostName` property.
          *
          * @param data - The EWS request.
          * @param callback - When the method completes, the function passed in the `callback` parameter is called with a single parameter,
          *                   `asyncResult`, which is an `Office.AsyncResult` object. The XML response of the EWS request is provided as a string
-         *                   in the `asyncResult.value` property. In Outlook on the web, on Windows (starting in Version 2303 (Build 16225.10000)),
-         *                   and on Mac (starting in Version 16.73 (23042601)), if the response exceeds 5 MB in size, an error message is returned
-         *                   in the `asyncResult.error` property. In earlier versions of the Outlook desktop client, an error message is returned if
-         *                   the response exceeds 1 MB in size.
+         *                   in the `asyncResult.value` property. In Outlook on the web, on classic Windows (starting in Version 2303 (Build 16225.10000)),
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:15802]:
```suggestion
         *                   in the `asyncResult.value` property. In Outlook on the web, in classic Outlook on Windows (starting in Version 2303 (Build 16225.10000)),
```

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -17970,9 +18045,10 @@ declare namespace Office {
          *
          * **Important**:
          *
-         * - In Outlook on the web, the reply form is displayed as a pop-out form in the 3-column view and a pop-up form in the 2-column or 1-column view.
+         * - In Outlook on the web and {@link https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627 | new Outlook on Windows},
+         * the reply form is displayed as a pop-out form in the 3-column view and a pop-up form in the 2-column or 1-column view.
          *
-         * - If any of the string parameters exceed their limits, `displayReplyForm` throws an exception.
+         * - If any of the string parameters exceed their limits, `displayReplyAllForm` throws an exception.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:18051]: good catch :)

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -11281,10 +11293,11 @@ declare namespace Office {
...
          *        `asyncContext`: Developers can provide any object they wish to access in the callback function.
-         *        `coercionType`: If text, the current style is applied in Outlook on the web and on Windows.
+         *        `coercionType`: If text, the current style is applied in Outlook on the web and on Windows (classic and
+         *                      {@link https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627 | new}.
          *                      If the field is an HTML editor, only the text data is inserted, even if the data is HTML.
-         *                      If html and the field supports HTML (the subject doesn't), the current style is applied in Outlook on the web and the
-         *                      default style is applied in Outlook on desktop clients.
+         *                      If the data is HTML and the field supports HTML (the subject doesn't), the current style is
+         *                      applied in Outlook on the web and new Outlook on Windows. The default style is applied in classic Outlook on Windows.
```
[the remark this answers is not in the document]
Comment (samantharamon) [reply types/office-js-preview/index.d.ts:11300]: Thanks for catching this. Added the Mac scenario.

**Review by Rick-Kirkham — APPROVED** [review]

File: types/office-js-preview/index.d.ts [hunk 4]
```diff
@@ -11293,14 +11292,14 @@ declare namespace Office {
...
-         *                      {@link https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627 | new}.
-         *                      If the field is an HTML editor, only the text data is inserted, even if the data is HTML.
-         *                      If the data is HTML and the field supports HTML (the subject doesn't), the current style is
-         *                      applied in Outlook on the web and new Outlook on Windows. The default style is applied in classic Outlook on Windows.
-         *                      If the field is a text field, an `InvalidDataFormat` error is returned.
-         *                      If `coercionType` is not set, the result depends on the field: if the field is HTML then HTML is used;
-         *                      if the field is text, then plain text is used.
+         *        `coercionType`: If text, the current style is applied in Outlook on the web, on Windows (classic and
+         *        {@link https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627 | new}, and on Mac.
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:11296]: Here and other instances...

```suggestion
         *        {@link https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627 | new}), and on Mac.
```

File: types/office-js-preview/index.d.ts [hunk 5]
```diff
@@ -11293,14 +11292,14 @@ declare namespace Office {
...
          *        `asyncContext`: Developers can provide any object they wish to access in the callback function.
-         *        `coercionType`: If text, the current style is applied in Outlook on the web and on Windows (classic and
-         *                      {@link https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627 | new}.
-         *                      If the field is an HTML editor, only the text data is inserted, even if the data is HTML.
-         *                      If the data is HTML and the field supports HTML (the subject doesn't), the current style is
-         *                      applied in Outlook on the web and new Outlook on Windows. The default style is applied in classic Outlook on Windows.
-         *                      If the field is a text field, an `InvalidDataFormat` error is returned.
-         *                      If `coercionType` is not set, the result depends on the field: if the field is HTML then HTML is used;
-         *                      if the field is text, then plain text is used.
+         *        `coercionType`: If text, the current style is applied in Outlook on the web, on Windows (classic and
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:11295]: Consider reordering to "new and classic" for consistency. Here and other instances

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**